src/screens/WebView/TermsAndCondition.js

```diff
@@ -3,34 +3,34 @@ import {ActivityIndicator, StyleSheet, View, SafeAreaView} from 'react-native';
 
 import {WebView} from 'react-native-webview';
 import Header from '../../components/Header';
-import { withInteractionsManaged } from '../../components/WithInteractionManaged';
+import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import {COLORS} from '../../utils/theme';
 
 const TermsAndCondition = ({navigation}) => (
-    <View style={styles.container}>
-      <SafeAreaView>
+  <View style={styles.container}>
+    <SafeAreaView>
       <Header title="Terms & Condition" onPress={() => navigation.goBack()} />
-
-      </SafeAreaView>
-      <WebView
-        source={{uri: 'https://www.lipsum.com/feed/html'}}
-        showsVerticalScrollIndicator={false}
-        startInLoadingState={true}
-        renderLoading={() => (
-          <View style={styles.content}>
-            <ActivityIndicator size="large" color="#00ADB5" />
-          </View>
-        )}
-      />
-    </View>
-  );
+    </SafeAreaView>
+    <WebView
+      source={{uri: 'https://www.lipsum.com/feed/html'}}
+      showsVerticalScrollIndicator={false}
+      startInLoadingState={true}
+      renderLoading={() => (
+        <View style={styles.content}>
+          <ActivityIndicator size="large" color={COLORS.holyTosca} />
+        </View>
+      )}
+    />
+  </View>
+);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been changed from a hardcoded value to a constant from the theme file. This is a good practice as it improves maintainability and consistency across the application.
```diff
- <ActivityIndicator size="large" color="#00ADB5" />
+ <ActivityIndicator size="large" color={COLORS.holyTosca} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->